README.md

```diff
@@ -1,16 +1,23 @@
-# Faded Text
+<p align="center">
+<img src="./docs/images/cover.png" height="220" />
+</p>
 
-[![Build Status](https://shields.io/github/actions/workflow/status/surfstudio/surf-flutter-faded-text/main.yml?logo=github&logoColor=white)](https://github.com/surfstudio/surf-flutter-faded-text)
-[![Pub Version](https://img.shields.io/pub/v/faded_text?logo=dart&logoColor=white)](https://pub.dev/packages/faded_text)
-[![Pub Likes](https://badgen.net/pub/likes/faded_text)](https://pub.dev/packages/faded_text)
-[![Pub popularity](https://badgen.net/pub/popularity/faded_text)](https://pub.dev/packages/faded_text/score)
-![Flutter Platform](https://badgen.net/pub/flutter-platform/faded_text)
+## Overview 🐄 
 
-## Description
+By default, text fade effects occur in the bottom-up direction if the content contains multiple lines. But sometimes it is necessary to make the fade effect from left to right, as it is done for single-line text in Flutter.
+This package solves this problem. It allows you to insert fade when wrap text on overflow from left to right.
 
-Faded Text is a package that allows you to create a fading text in case of overflow.
+If the text is overflow, it will look like this:
 
-## Installation
+<p align="center">
```

Comment:
Perhaps it would be better to demonstrate the difference between the default option and the library version? Place the engine's version on the left and the demonstration of the library's work on the right.

It's not very clear why there is a separate black and white version?

Reply:
Resolve in #3

Reply:
Resolve in #3